hl13/code/modules/projectiles/boxes_magazines/ar2.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/ammo_box/magazine/hl13/ar2
```

Comment:
same as for the ar2 comment (just saw this after)

Reply:
All organized now